ui/src/components/view/DetailsTab.vue

```diff
@@ -35,7 +35,7 @@
         </div>
         <div v-else-if="$route.meta.name === 'computeoffering' && item === 'rootdisksize'">
           <div>
-            {{ parseFloat( resource.rootdisksize / (1024.0 * 1024.0 * 1024.0)).toFixed(1) }} GB
+            {{ resource.rootdisksize }} GB
```

Comment:
@shwstppr thanks for pointing out the change necessary to keep UI aligned with API.
I believe this will fix it, please let me know if I am missing anything.